changelogs/current.yaml

```diff
@@ -226,6 +226,10 @@ new_features:
   change: |
     Added :ref:`bypass_overload_manager <envoy_v3_api_field_config.listener.v3.Listener.bypass_overload_manager>`
     to bypass the overload manager for a listener. When set to true, the listener will not be subject to overload protection.
+- area: dns_filter
+  change: |
+    Added support wildcard names in :ref:`inline_dns_table
```

Comment:
Can we change to this: Added support for wildcard resolving when DNS filter is working in server mode?